style.css

```diff
@@ -0,0 +1,74 @@
+.header {
+  display: flex;
+  color: white;
+  background-color: #3c3a39;
+  height: 10vh;
+  flex-direction: row;
+  align-items: center;
+}
+
+.name {
+  position: relative;
+  left: 40px;
+  font-family: 'Roboto', sans-serif;
+  font-size: 22px;
+  color: #fff;
+}
+
+.menu {
+  position: relative;
+  left: 310px;
+}
```

Comment:
Kindly check the width of the elements inside the main container, right now you have white space and there is a scroll on the horizontal axis.
Let's delete the 8px margin around the body element. 
<img width="844" alt="Screenshot 2022-12-01 at 1 25 17 AM" src="https://user-images.githubusercontent.com/25887253/204923332-9e3bae90-5f34-4698-88c0-56662f148823.png">
When you fix the nav and all the elements' sizes like  `<p>` element, it will be fixed. Right now, it looks like this below.
<img width="386" alt="Screenshot 2022-12-01 at 1 02 43 AM" src="https://user-images.githubusercontent.com/25887253/204923663-63f9eb2d-e9e0-45fa-b1ef-7c5c9224129e.png">